taichi/program/kernel.cpp

```diff
@@ -247,12 +239,6 @@ void Kernel::set_arg_nparray(int i, uint64 ptr, uint64 size) {
   TI_ASSERT_INFO(args[i].is_nparray,
                  "Assigning numpy array to scalar argument is not allowed");
 
-  ActionRecorder::get_instance().record(
-      "set_kernel_arg_ext_ptr",
-      {ActionArg("kernel_name", name), ActionArg("arg_id", i),
-       ActionArg("address", fmt::format("0x{:x}", ptr)),
-       ActionArg("array_size_in_bytes", (int64)size)});
-
```

Comment:
There are still people using information like `set_kernel_arg_int64`. Can we keep these for now?

Reply:
Well, it doesn't even include `dtype`... They are just spams in our compile-only no-launch action parser, both in CC and [数据删除] in my pratice...

Reply:
But there are still people depending on this. I don't think we should remove them immediately. Maybe a better solution is to mark them as a different type of message.

Reply:
I thought the feature is `purposely undocumented` and used by only you and me and the [数据删除]? Also did you confirmed this doesn't harm performance? IMO we'd remove all the run-time action output iapr, since only compile-time kernel information are required for reconstructing a Taichi program outside, e.g., this PR, did that easily.